pandas/core/arrays/numeric.py

```diff
@@ -130,3 +158,16 @@ def _arith_method(self, other, op):
             )
 
         return self._maybe_mask_result(result, mask, other, op_name)
+
+    def _apply(self, func: Callable, **kwargs) -> "NumericArray":
+        values = self._data[~self._mask]
```

Comment:
I don't think you actually need to subset the `_data` with the mask in this case, as "round" should work on all values, and I can't think of a case where it would error by being called on the "invalid" values hidden by the mask.

Of course, if many values are masked, we might be calculating `round` on too many values. But doing the filter operation / copy also takes time. Maybe something to time both ways.